Gems/EMotionFX/Code/EMotionFX/Source/MorphTargetStandard.cpp

```diff
@@ -169,28 +169,27 @@ namespace EMotionFX
 
     // apply the relative transformation to the specified node
     // store the result in the position, rotation and scale parameters
-    void MorphTargetStandard::ApplyTransformation(ActorInstance* actorInstance, uint32 nodeIndex, AZ::Vector3& position, AZ::Quaternion& rotation, AZ::Vector3& scale, float weight)
+    void MorphTargetStandard::ApplyTransformation(ActorInstance* actorInstance, size_t nodeIndex, AZ::Vector3& position, AZ::Quaternion& rotation, AZ::Vector3& scale, float weight)
     {
         // calculate the normalized weight (in range of 0..1)
         const float newWeight = MCore::Clamp<float>(weight, mRangeMin, mRangeMax); // make sure its within the range
         const float normalizedWeight = CalcNormalizedWeight(newWeight); // convert in range of 0..1
 
         // calculate the new transformations for all nodes of this morph target
-        const uint32 numTransforms = mTransforms.GetLength();
-        for (uint32 i = 0; i < numTransforms; ++i)
+        for (const Transformation& mTransform : mTransforms)
```

Comment:
mTransform -> transform
This seems to have happened on multiple places.

Reply:
I fixed all the places where the loop variable was named after the member variable that it was iterating over. See 
8a286ee.